platform/lf_nrf52_support.c

```diff
@@ -104,6 +111,18 @@ void calculate_epoch_offset() {
  */
 uint32_t _lf_previous_timer_time = 0u;
 
+void lf_timer_event_handler(nrf_timer_event_t event_type, void *p_context)
+{
+    // interrupted by timer first
+    if (event_type == NRF_TIMER_EVENT_COMPARE2)
```

Comment:
Add comment. What is the meaning of NRF_TIMER_EVENT_COMPARE2? Also, add curly braces.

Reply:
added